src/modules/bank/injectBank.js

```diff
@@ -1,16 +1,120 @@
-import ajaxifyBank from './ajaxifyBank';
+import ErrorMessage from './ErrorMessage.svelte';
+import createDiv from '../common/cElement/createDiv';
+import daBankDeposit from '../_dataAccess/daBankDeposit';
+import daBankWithdraw from '../_dataAccess/daBankWithdraw';
+import querySelector from '../common/querySelector';
+import querySelectorArray from '../common/querySelectorArray';
 
-export default function injectBank() {
-  ajaxifyBank({
-    headSelector: '#pCC h2',
-    headText: 'Bank',
-    appLink: true,
-    depoPos: 1,
-    balPos: 0,
-    data: {
-      cmd: 'bank',
-      subcmd: 'transaction',
-    },
-    initWithdraw: '',
+function preventDefault(func) {
+  return (e) => { e.preventDefault(); func(); };
+}
+
+function parseFormattedInt(str) {
+  return parseInt(str.replace(/,/g, ''), 10);
+}
+
+function formatInt(i) {
+  return i.toLocaleString('En-US');
+}
+
+function toggleSubmit(b) {
+  const depositButton = querySelector('input[value="Deposit"]');
+  const withdrawButton = querySelector('input[value="Withdraw"]');
+
+  if (b === undefined) {
+    // eslint-disable-next-line no-param-reassign
+    b = !withdrawButton.disabled;
+  }
+
+  withdrawButton.disabled = b;
+  depositButton.disabled = b;
+}
+
+function updateInputs(newMaxDeposit, newBalance) {
+  const depositInput = querySelector('#deposit_amount');
+  const withdrawInput = querySelector('#withdraw_amount');
+  depositInput.value = Math.min(parseInt(depositInput.value, 10), newMaxDeposit);
+  withdrawInput.value = Math.min(parseInt(withdrawInput.value, 10), newBalance);
+}
+
+function showError(message) {
+  const pCC = querySelector('#pCC');
+  const target = createDiv();
+  pCC.prepend(target);
+  return new ErrorMessage({
```

Comment:
we do not want to be instantiating a new component every time we get an error. instantiate it once, retain a handle to it, update the message as required or hide it.